### ISSUE TITLE
Windows Terminal not snapping correctly with the keys `Win+Left` and `Win+Right`

### ISSUE THREAD

**Issue by gbr000** [microsoft/terminal#14847]
### Windows Terminal version

1.17.1023

### Windows build number

10.0.19045.0

### Steps to reproduce

1. Open the Terminal
2. Press Win+Left
3. Press Win+Right

### Expected Behavior

2. snap to the LEFT side of the screen
3. snap to the RIGHT side of the screen

### Actual Behavior

2. it snapped to the right side of the screen all in a crazy fashion
3. it snapped to the left side of the screen all in a crazy fashion

~sometimes it doesn't snap at all, and sometimes this problem doesn't always occur.

I think (but not sure) it only does happen when you leave the terminal opened for hours!

~The problem is recurrent, it happens constantly and in random times

**Comment by gbr000** [comment]
Strangely, press `Win+down`, then again to minimize that window,(the window will be hided and without releasing `Win` all the time), play aroun with that `Win` key in the minimized state, go LEFT then go up again. Now the snap is inverted.

*NOTE: I could only reproduce this once on my own, but the program automatically starts in the mode when he wanted.

So I could indentify part of the problem but not the cause!

**Comment by gbr000** [comment]
I cannot reproduce it all the time but it happens when I don't want it. (I'm investigating the reason why)

Most of the times happened after leaving the Terminal opened for hours like 3 to 6 hours (and maybe closing the lid and reopning it after?)

I just updated the Windows 10, maybe it was a Windows problem rather then Terminal, for now there's no problems. I'm thinking about upgrade to Win 11.

*I cannot reproduce this when I want, basically when you press `Win+Left` the window goes right, it gets inversed and the windows a confused behavior (What I have noticed is that when the windows is at that behavior it gets a little off the scene and is inverted but not in the normal way). And it's possible to fix that inversion by pressing `Win+arrow keys` many times and it fixes, also in that state it's possible to go back to inversion. But to get to that state I cannot!

state = time when the bug occurs!
